crates/steel-repl/src/repl.rs

```diff
@@ -34,7 +34,7 @@ fn display_help() {
         "
         :time       -- toggles the timing of expressions
         :? | :help  -- displays help dialog
-        :quit       -- exits the REPL
+        :q | :quit       -- exits the REPL
         :pwd        -- displays the current working directory
```

Comment:
```suggestion
        :q | :quit  -- exits the REPL
        :pwd        -- displays the current working directory
```

This should probably be alligned